src/fhirtypes/InstanceDefinition.ts

```diff
@@ -2,6 +2,7 @@ import cloneDeep = require('lodash/cloneDeep');
 import { Meta } from './specialTypes';
 import { HasId } from './common';
 import { applyMixins } from '../utils';
+import { isNull } from 'util';
```

Comment:
As GitHub has pointed out, 'isNull' is defined but never used.

Reply:
Ah, dang.  I remember that happening.  VS Code auto-completed something to `isNull`, which I did not intend -- so I fixed it, but hadn't realized that VS Code also auto-added an import for it.  Doh.